src/commands/cmd_query.c

```diff
@@ -96,7 +96,7 @@ void Graph_Query(void *args) {
 	QueryCtx_SetResultSet(result_set);
```

Comment:
IMO we may as well move the `QueryCtx_SetResultSet` call into `NewResultSet`, then have no local ResultSet variable at all in the various commands.

Reply:
I prefer to avoid this "hiding" I think being explicit here is preferable, 
`NewResultSet` should return a `result-set` object, let the caller decide what to do with it.